### ISSUE TITLE
Bug with GetProfilesAsync

### ISSUE THREAD

**Issue by superjacobl** [drasticactions/FishyFlip#1]
In GetProfilesAsync, having the query part of the url just be "?actors=actor1.did,actor2.did,actor3.did" does not work and results in an error that says actor/0 must be a did or handle.
After discovering this, I, after a little while, found a solution. Bluesky's server wants the query part to be "?actors=actor1.did&actors=actor2.did&actors=actor3.did" for some reason. Using the httpclient from a ATProtocol object, I was able to confirm that after sending a request with the query part as that, bluesky returns the list of profiles.

**Comment by drasticactions** [comment]
When I originally wrote that code, I believed that's what ATProtocol expected as values. Either that changed since I wrote it, or it was always broken, and I didn't catch it, or both.

https://github.com/drasticactions/FishyFlip/commit/8ab707525ab09e663886979368dc094258b473ca

I've switched out all uses of `string.join` with commas and replaced it with key-value pairs, and wrote tests for them. 

https://www.nuget.org/packages/FishyFlip/1.1.35-alpha

Let me know if that works for you.
